workflow/rocoto/workflow_tasks.py

```diff
@@ -466,21 +466,9 @@ def atmanlfinal(self):
 
     def aeroanlinit(self):
 
-        dump_suffix = self._base["DUMP_SUFFIX"]
-        dmpdir = self._base["DMPDIR"]
-        atm_hist_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_HISTORY_TMPL"], {'RUN': 'gdas'})
-        dump_path = self._template_to_rocoto_cycstring(self._base["COM_OBSDMP_TMPL"],
-                                                       {'DMPDIR': dmpdir, 'DUMP_SUFFIX': dump_suffix})
-
         deps = []
-        dep_dict = {'type': 'metatask', 'name': 'gdaspost', 'offset': '-06:00:00'}
-        deps.append(rocoto.add_dependency(dep_dict))
-        data = f'{atm_hist_path}/gdas.t@Hz.atmf009.nc'
-        dep_dict = {'type': 'data', 'data': data, 'offset': '-06:00:00'}
-        deps.append(rocoto.add_dependency(dep_dict))
         dep_dict = {'type': 'task', 'name': f'{self.cdump}prep'}
```

Comment:
Why the changes to the aerosol analysis dependencies?

Reply:
because they are not needed. And I am breaking my own rule of not adding unrelated code updates.
Do you want me to pull these out in its own PR?

Reply:
I'm fine leaving the changes as long as we are sure the dependencies are not needed. I guess `prep` already has the same ones?

Reply:
yeah the dependencies are 1) the previous forecast completing and 2) observations being available. `prep` should take care of those I think